### PR TITLE
#74 - Use eo-yaml library

### PR DESCRIPTION
**Pull request by genryxy**
Partf of #74 
Used `eo-yaml` instead of `snakeyaml` since the indentations for the sequences were incorrect. Added simple test. Further I'll add a few more tests.

### CODE REVIEW

**Comment by olegmoz** [comment]
@genryxy the PR is massive and hard to review. Could you please try to split it into 2-3 PRs of lesser size?

**Comment by genryxy** [comment]
@olegmoz sorry, reduce the size of PR

**Comment by genryxy** [comment]
@olegmoz Also, I updated description of the issue to make my intent more clear. Now the library is changed and the first case (i. e. `entries` node  isn't represented in the `index.yaml`) is covered.

File: src/main/java/com/artipie/helm/IndexYaml.java [hunk 1]
```diff
@@ -98,83 +91,129 @@ public Completable update(final TgzArchive arch) {
...
-                    Collections.singleton(String.format("%s%s", this.base, tgz.name()))
-                )
+        return resyaml;
+        // @checkstyle MethodBodyCommentsCheck (8 lines)
+        // @todo #32:30min Create a unit test for digest field
+        //  One of the fields Index.yaml require is "digest" field. The test should make verify
+        //  that field has been generated correctly.
+        // @todo #32:30min Create a unit test for urls field
+        //  One of the fields Index.yaml require is "urls" field. The test should make verify
+        //  that field has been generated correctly.
```
Comment (olegmoz) [inline src/main/java/com/artipie/helm/IndexYaml.java:157]: @genryxy do not understand what is the reason to place puzzles here, I think it more appropriate to add these puzzles to JavaDoc section of method or class that requires test coverage

**Review by olegmoz — CHANGES_REQUESTED** [review]
@genryxy thanks, please see one minor comment regarding formatting

**Comment by genryxy** [comment]
@olegmoz thanks, they were inside the method. Moved them to the Javadoc section.

**Review by olegmoz — APPROVED** [review]
@genryxy thanks, merging

**Comment by 0crat** [comment]
Job `gh:artipie/helm-adapter#75` is not assigned, can't get performer

**Pull request merged**